node/src/reactor/main_reactor.rs

```diff
@@ -1391,38 +1391,13 @@ impl MainReactor {
             return effects;
         }
 
-        // Set the current switch block only after the block is marked complete.
         // We *always* want to initialize the contract runtime with the highest complete block.
         // In case of an upgrade, we want the reactor to hold off in the `Upgrading` state until
         // the immediate switch block is stored and *also* marked complete.
         // This will allow the contract runtime to initialize properly (see
         // [`refresh_contract_runtime`]) when the reactor is transitioning from `CatchUp` to
         // `KeepUp`.
-        if state.is_marked_complete() {
-            debug!(
-                "MetaBlock: marked complete: {} {}",
-                block.height(),
-                block.hash(),
-            );
```

Comment:
This debug message is now gone. This is probably perfectly fine, because we'll have the `MetaBlock: update gossip state` logged, but just wanted to drag you attention to it for double check.

Reply:
Good point, I'll add it back since it helps with debugging.

Reply:
Added in https://github.com/casper-network/casper-node/pull/3971/commits/2b4c9ae04b1ad1c97b873fabb58367cc72e32144